sig-instrumentation/charter.md

```diff
@@ -0,0 +1,55 @@
+# SIG Instrumentation Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+Owns best practices for cluster observability through metrics, logging, and
+events across all Kubernetes components and development of relevant components.
+
+### In scope
+
+Coordinating metric requirements of different SIGs for other components through finding common APIs.
+
+#### Code, Binaries and Services
+
+[List of subprojects](https://github.com/kubernetes/community/tree/master/sig-instrumentation#subprojects)
+
+- Components required for any Kubernetes cluster in regards to observability.
```

Comment:
What are those components? Are they the subprojects or something else? Can you link to them?